src/templates/csr/capi-multiple-paidfor/index.svelte

```diff
@@ -0,0 +1,153 @@
+<script context="module" lang="ts">
+	export const cdn = 'https://i.guim.co.uk/img/media/';
+	export const api =
+		'https://api.nextgen.guardianapps.co.uk/commercial/api/capi-multiple.json';
+	import '$templates/components/fonts/Sans.css';
+</script>
+
+<script lang="ts">
+	import type { GAMVariable } from '$lib/gam';
+	import type { CapiCardOverride } from '$lib/types/capi';
+
+	import Sponsor from '$templates/components/Sponsor.svelte';
+	import CapiCard from '$templates/components/CapiCard.svelte';
+	import PaidForHeader from '$templates/components/PaidForHeader.svelte';
+	import { addTrackingPixel, isValidReplacedVariable } from '$lib/gam';
+	import Resizer from '$templates/components/Resizer.svelte';
+	import { retrieveCapiData, addHeadlineKicker } from '$lib/capiMultiple';
+
+	export let SeriesUrl: GAMVariable;
+	export let ComponentTitle: GAMVariable;
+	export let Article1Headline: GAMVariable;
+	export let Article1Image: GAMVariable;
+	export let Article1URL: GAMVariable;
+	export let Article1Kicker: GAMVariable;
+	export let Article2Headline: GAMVariable;
+	export let Article2Image: GAMVariable;
+	export let Article2URL: GAMVariable;
+	export let Article2Kicker: GAMVariable;
+	export let Article3Headline: GAMVariable;
+	export let Article3Image: GAMVariable;
+	export let Article3URL: GAMVariable;
+	export let Article3Kicker: GAMVariable;
+	export let Article4Headline: GAMVariable;
+	export let Article4Image: GAMVariable;
+	export let Article4URL: GAMVariable;
+	export let Article4Kicker: GAMVariable;
+	export let Trackingpixel: GAMVariable;
+
+	let cards: CapiCardOverride[] = [
+		{
+			headline: Article1Headline,
+			image: Article1Image,
+			url: Article1URL,
+			kicker: Article1Kicker,
+		},
+		{
+			headline: Article2Headline,
+			image: Article2Image,
+			url: Article2URL,
+			kicker: Article2Kicker,
+		},
+		{
+			headline: Article3Headline,
+			image: Article3Image,
+			url: Article3URL,
+			kicker: Article3Kicker,
+		},
+		{
+			headline: Article4Headline,
+			image: Article4Image,
+			url: Article4URL,
+			kicker: Article4Kicker,
+		},
+	];
+
+	const requestData = retrieveCapiData(cards, SeriesUrl);
+
+	if (isValidReplacedVariable(Trackingpixel)) addTrackingPixel(Trackingpixel);
+
+	let height: number = -1;
+</script>
+
+{#await requestData}
+	<h3>Loading Content for “{SeriesUrl}”</h3>
+{:then response}
+	{#await addHeadlineKicker(cards, response.articles)}
+		<p>Formatting cards...</p>
+	{:then formattedCards}
```

Comment:
I think it might be nicer to generate `cards` with a promise? and then await that? 

something like:
```js
const cardOverrides: CapiCardOverride[] = [
//...
];

const getCards = retrieveCapiData(cardOverrides, SeriesUrl).then((response) => addHeadlineKicker(cardOverrides, response.articles));

{#await getCards}
{:then cards}
//
{/await}
```